distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -102,10 +50,101 @@ export class Chart extends React.Component {
       return chart;
     }
   }
+
+  setMinMaxState(showOutliers) {
+    const pointsMetaLength = this.pointsMeta.length;
+    let minIndex, maxIndex;
+
+    if (showOutliers) {
```

Comment:
The math in the `else` would account for it but we could avoid it for smaller data sets by doing `|| pointsMetaLength < 100`. Treat this as a nitpick - it's not necessary.